### PR TITLE
Modify for group: To extend index by removing useless link

### PR DESCRIPTION
**Pull request by Casmo888**

### CODE REVIEW

**Comment by Casmo888** [comment]
Please review the newest one, thanks!

File: group/templates/group/viewall.html [hunk 1]
```diff
@@ -22,25 +22,9 @@
...
-  <link rel="stylesheet" href="/css/magnific-popup.css">
-  <link rel="stylesheet" href="/css/mfp-fade.css">
-  <script src="/js/jquery.magnific-popup.js"></script>
-  <script src="/js/popup-setting.js"></script>
-</head>
-
+{% extends "index/base.html" %} 
+{% load static %} 
+{% block body_block %}
 <body>
```
Comment (nickelcheng) [inline group/templates/group/viewall.html]: You deleted the corresponding end tag, but didn't delete this?

**Comment by Casmo888** [comment]
Modify followed comment, thanks for review

**Comment by nickelcheng** [comment]
I have no problem with this PR. Does @CHIAHSINHOU or @bruce3557 have any suggestion?

File: group/templates/group/viewall.html [hunk 2]
```diff
@@ -22,100 +22,77 @@
...
+        {% endif %} {% if list_type != "runContest" %}
+        <td>{{ archive.time }}</td>
+        {% endif %} {% if list_type == "announce" %}
+        <td>
+          <a href="" class="btn btn-sm btn-default" data-toggle="modal" 
+             data-target="#annowenceContentModal{{archive.id}}" style="float:right">
+            Detail</a>
+          <!--1. Modal for Edit Annowence button-->
+          <div class="modal fade" id="annowenceContentModal{{archive.id}}" 
+               tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
```
Comment (nickelcheng) [inline group/templates/group/viewall.html]: Please use a meaningful name instead of myXXX

**Comment by Casmo888** [comment]
After testing, decided to remove the label because id is already to distinguish the modal.
Thanks for reviewing!

**Comment by nickelcheng** [comment]
Thanks, going to merge.

**Pull request merged**